### ISSUE TITLE
[BUG] Game boot in handheld will disable LunaKit-Rendering

### ISSUE THREAD

**Issue by MonsterDruide1** [Amethyst-szs/smo-lunakit#29]
**Describe the bug**
Starting the game in handheld mode (not being docked), then trying to open the menu will yield nothing. In the server log, I can see that it's attempting to open it (alternating between `Active state Enabled`/`Active state Disabled`), but nothing is shown on the screen.

**To Reproduce**
1. Take the console from the dock
2. Launch the game
3. Open the menu, observe that nothing happens
4. Put the switch into the dock, while it should show the menu, and the menu will show up on TV screen
5. Take the switch out, and rendering will continue to work just fine
6. Closing and re-opening the menu will also still work, seems like only opening the menu for the first time is the issue.

**Screenshots**
Not helpful, just that the menu doesn't show up as if it wasn't opened

**Platform**
Nintendo Switch

**Comment by miteduk3** [comment]
I'm having the same problem, and I've found (what I'm calling) the best solution to this problem

Go online and find (or in the homebrew app store) 
"ReverseNX-tool" I thought I would never actually need it, but this thing helped solve this issue

Long story short, when you boot up reverseNX you can basically choose the games that are tricked into thinking they are in docked mode! And since it only works in docked mode, it works regardless! Ive also tested it, and no, this does not drain battery or push the switch harder than it should.

Basically, just set it to docked mode, and then just never change it (unless you want to) and you should always be able to open the menu

Your welcome :D

**Comment by ReznorsRevenge1** [comment]
I've found using reverseNX doesn't change anything, and I'm having the same problem (LunaKit interface only works if game was once in docked mode).
